e2e_test/source_inline/kafka/consumer_group.slt

```diff
@@ -0,0 +1,101 @@
+# Note: control substitution on will force us to use "\\n" instead of "\n" in commands
+control substitution on
+
+# Note either `./risedev rpk` or `rpk` is ok here.
+# risedev-env contains env var RPK_BROKERS, which is read by rpk
+system ok
+rpk topic create test_consumer_group -p 3
+
+system ok
+cat <<EOF | rpk topic produce test_consumer_group -f "%p %v\\n" -p 0
```

Comment:
Can we also extract this pipeline into a single command?

Reply:
Can you suggest an improvement? I feel the command is simple enough.

Reply:
For instance, if we are going to adopt the same `"%p %v\n"` format in several places...

Reply:
We will also use other formats, especially "%k %v\n" I guess